app/models/developers/title.rb

```diff
@@ -0,0 +1,55 @@
+module Developers
+  class Title
+    private attr_reader :query
+
+    def initialize(query)
+      @query = query
+    end
+
+    def title
+      components = []
+
+      # i18n-tasks-use t("developers.title.hire")
+      components << i18n("hire")
+
+      components << role_level if role_level
+
+      # i18n-tasks-use t("developers.title.freelance")
+      components << i18n("freelance") if freelance?
+
+      # i18n-tasks-use t("developers.title.ruby_on_rails_developers")
+      components << i18n("ruby_on_rails_developers")
+
+      # i18n-tasks-use t("developers.title.in_country")
+      components << i18n("in_country", country:) if country
+
+      components.join(" ")
+    end
+
+    private
+
+    def i18n(key, options = {})
```

Comment:
I usually name this method `t` similar to how it’s done in the views

Reply:
Also I would do **options so that

Reply:
Sadly `i18n-tasks` thinks that using `t` means we are using the view helper and gets all confused.